h5p-bildetema/src/components/CollectionsController/CollectionController.tsx

```diff
@@ -19,9 +21,10 @@ type CollectionControllerProps = {
 const CollectionController = ({
   rtl,
 }: CollectionControllerProps): JSX.Element => {
-  const { collection } = useParams();
+  const h5pInstance = useH5PInstance<H5PWrapper>();
   const langCode = useCurrentLanguageCode();
   const words = useSelectedWords();
+  const { collection } = useParams();
```

Comment:
You should add this const { collection } = useParams<{ collection: string }>();  for  type safety.

Reply:
Suggestion added